infra/terraform/test-org/github/branch.tf

```diff
@@ -44,7 +53,35 @@ data "github_team" "cft-admins" {
 resource "github_branch_protection" "master" {
   for_each      = data.github_repository.repo
   repository_id = each.value.node_id
-  pattern       = "master"
+  pattern       = each.value.default_branch
+
+  required_pull_request_reviews {
```

Comment:
nit: since we can't do provider for_each, as a follow up it may make sense to pull out a base repo config (labels, branch protection, files etc) as a module that we can then instantiate twice here passing in two explicit providers.

Reply:
Curiously labels don't need separate providers, but I abstracted both branch_protection and a generic repo_file as modules.